stock_reception_screen_measuring_device/migrations/14.0.2.0.0/post-migration.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def populate_new_field(env):
+    records_to_update = (
+        env["stock.reception.screen"].search([("current_step", "!=", "done")]).exists()
+    )
+    _logger.info(
+        "Set smaller_package_has_missing_dimensions on ongoing reception screens"
+    )
+    records_to_update._compute_smaller_package_has_missing_dimensions()
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    env = api.Environment(cr, SUPERUSER_ID, {})
```

Comment:
You can use `@openupgrade.migrate()` as decorator to get an environment as parameter instead of `cr` (not blocking)